src/main/java/datastax/astra/migrate/AbstractJobSession.java

```diff
@@ -95,13 +96,21 @@ protected AbstractJobSession(CqlSession sourceSession, CqlSession astraSession,
         }
 
         String customWriteTimeStr =
-                Util.getSparkPropOr(sc, "spark.target.custom.writeTime", "0");
-        if (null != customWriteTimeStr && customWriteTimeStr.trim().length() > 1 && StringUtils.isNumeric(customWriteTimeStr.trim())) {
-            customWritetime = Long.parseLong(customWriteTimeStr);
+                Util.getSparkPropOr(sc, "spark.target.writeTime.fixedValue", "0");
+        if (null != customWriteTimeStr && customWriteTimeStr.trim().length() > 0 && StringUtils.isNumeric(customWriteTimeStr.trim())) {
```

Comment:
Could we please use the lang3 apache commons here to further simply it as,
```suggestion
        if (StringUtils.isNotBlank(customWriteTimeStr) && StringUtils.isNumeric(customWriteTimeStr.trim())) {
```